fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -1756,6 +1762,80 @@ private Bundle processEntriesForMethod(Bundle requestBundle, Bundle responseBund
         }
     }
 
+    /**
+     * Processes a request entry with a request method of Patch.
+     *
+     * @param requestEntry
+     *            the request bundle entry
+     * @param responseEntry
+     *            the response bundle entry
+     * @param responseIndexAndEntries
+     *            the hashmap containing bundle entry indexes and their associated response entries
+     * @param entryIndex
+     *            the bundle entry index of the bundle entry being processed
+     * @param localRefMap
+     *            the map of local references to external references
+     * @param requestURL
+     *            the request URL
+     * @param absoluteUri
+     *            the absolute URI
+     * @param requestDescription
+     *            a description of the request
+     * @param initialTime
+     *            the time the bundle entry processing started
+     * @throws Exception
+     */
+    private void processEntryforPatch(Entry requestEntry, Entry responseEntry, Map<Integer, Entry> responseIndexAndEntries, Integer entryIndex,
+        Map<String, String> localRefMap, FHIRUrlParser requestURL, String absoluteUri, String requestDescription, long initialTime)
+        throws FHIROperationException {
+        FHIRRestOperationResponse ior = null;
+        int resourceIdIndex = 7;
+        int resourceTypeIndex = 6;
+        String resourceType = extractValueFromUrl(requestEntry.getRequest().getUrl().getValue(), resourceTypeIndex);
+        String resourceId = extractValueFromUrl(requestEntry.getRequest().getUrl().getValue(), resourceIdIndex);
```

Comment:
This makes assumptions about the incoming URL that I'd rather not make.  FHIR Server operators may chose to use whatever path they want for the `[base]` of the FHIR Server URL.